admin/acceptance.py

```diff
@@ -222,7 +232,7 @@ class RunOptions(Options):
          'One of fedora-20.'],
```

Comment:
I wouldn't bother. We'll soon be adding Ubuntu and Centos so it'll make sense then.